src/lib/media/manip.ts

```diff
@@ -203,11 +209,36 @@ function joinPath(a: string, b: string) {
   return a + '/' + b
 }
 
-function normalizePath(str: string, allPlatforms = false): string {
-  if (isAndroid || allPlatforms) {
-    if (!str.startsWith('file://')) {
-      return `file://${str}`
-    }
+function normalizePath(str: string): string {
+  if (!str.startsWith('file://')) {
+    return `file://${str}`
   }
   return str
 }
```

Comment:
There's no need to make a distinction between adding `file://` and not on iOS. Both are supported (Android only supports adding paths with `file://`). We can simplify this logic by just always adding it.